src/newcharacter.cpp

```diff
@@ -2690,6 +2710,13 @@ tab_direction set_description( avatar &you, const bool allow_reroll,
                         you.mod_base_age( 1 );
                     }
                     break;
+                case char_creation::BLOOD:
+                    if( you.blood_type != AB_MINUS ) {
```

Comment:
Suggest you use an explicitly last enumerator here rather than assuming `AB_MINUS` is last.